forc-plugins/forc-client/src/op/deploy.rs

```diff
@@ -122,6 +132,70 @@ fn validate_and_parse_salts<'a>(
     Ok(contract_salt_map)
 }
 
+async fn deploy_chunked(
+    command: &cmd::Deploy,
+    compiled: &BuiltPackage,
+    salt: Salt,
+    signing_key: &SecretKey,
+    provider: &Provider,
+    pkg_name: &str,
+) -> anyhow::Result<(ContractId, Vec<ContractId>)> {
+    println_action_green("Splitting", &format!("{pkg_name} into chunks"));
+    let contract_chunks = split_into_chunks(compiled.bytecode.bytes.clone(), MAX_CONTRACT_SIZE)?;
+    let mut deployed_contracts = vec![];
+    println_action_green("Deploying", &format!("{pkg_name} chunks"));
+    let chain_info = provider.chain_info().await?;
+    let target = Target::from_str(&chain_info.name).unwrap_or(Target::testnet());
+    let contract_url = match target.explorer_url() {
+        Some(explorer_url) => format!("{explorer_url}/contract/0x"),
+        None => "".to_string(),
+    };
+    for contract_chunk in contract_chunks {
+        let chunk_id = contract_chunk.id();
+        let deployed_chunk = contract_chunk.deploy(provider, signing_key).await?;
+        let chunk_contract_id = deployed_chunk.contract_id();
+        println_action_green(
+            "Finished",
+            &format!("deploying chunk {chunk_id} for {pkg_name}: {chunk_contract_id}"),
+        );
+        deployed_contracts.push(deployed_chunk);
+    }
+    println_action_green("Deployed", &format!("{pkg_name} chunks"));
```

Comment:
nit: it would be nice to time the whole operation and show "Deploy X chunks in X ms"

Reply:
We don't have the `Deployed` prompt right now. We are following the same pattern as normal deplyoment in terms of the output.

Also deployment in chunks is much faster now since we removed unnecessary steps, such as cloning the chunks before each deployment to create contracts out of them as we have blobs and sdk support for blobs now. 

If you feel like this is still valuable, we can add it to the finished prompt though